app/src/main/java/com/example/weathertimeapp/mvp/view/WeatherTimeView.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.weathertimeapp.mvp.view
+
+import android.app.Activity
+import android.widget.ArrayAdapter
+import android.widget.AutoCompleteTextView
+import com.example.weathertimeapp.mvp.contracts.WeatherTimeContracts
+import com.example.weathertimeapp.mvp.view.base.ActivityView
+import kotlinx.android.synthetic.main.activity_main.activity_main_edit_text_city
+
+class WeatherTimeView(activity: Activity) : ActivityView(activity), WeatherTimeContracts.View {
+    private val autoCompleteTextView: AutoCompleteTextView = activity.activity_main_edit_text_city
+
+    override fun setCitiesListAdapter(cities: MutableList<String>) {
+        val adapter = activity?.applicationContext?.let { ArrayAdapter(it, android.R.layout.simple_list_item_1, cities) }
+        autoCompleteTextView.setAdapter(adapter)
```

Comment:
can you access this with synthetics?

Reply:
i'm using synthetics, i assign autoCompleteTextView with it at the beginning of the class, and then i use it on the function

Reply:
then it's necessary to use autoCompleteTextView instead of activity.activity_main_edit_text_city ?

Reply:
you are right, in this moment is not necessary because i just use it once.
removed to reduce unnecessary code, thanks